code/client/clrcore/External/World.cs

```diff
@@ -39,94 +39,98 @@ public override int GetDaysInMonth(int year, int month, int era)
 	}
 	enum ZoneID
 	{
-		AIRP,
```

Comment:
what happened here?

Reply:
I think that's VS automatically adjusting the spaces to what the other ones have because almost nothing actually changed in that list

Reply:
That's not it, I'm very unsure as to what happened because I see lines changed that I have not touched or see no change to at all regarding any spacing or whatever.

Reply:
I used diffchecker.com to see if that'd give me the same output, but it didn't, that one gave me the right output, github just doesn't for some reason

Reply:
I tried pasting in the original code and editing it on GitHub web, but that didn't seem to fix the issues unfortunately.